### ISSUE TITLE
#shared crashes pqtest.exe

### ISSUE THREAD

**Issue by bgribaudo** [microsoft/vscode-powerquery-sdk#113]
If my `.test.pq` file returns `#shared` as its output, pqtest.exe crashes. :-(

````
let
    Result = #shared
in
    Result
````

````
[2:10:01 PM]	[Error]	[Task exited abnormally] pqtest run-test pid(2780) exit(3762504530) stderr: 
Unhandled Exception: System.ArgumentException: .NET number values such as positive and negative infinity cannot be written as valid JSON.
   at System.Text.Json.ThrowHelper.ThrowArgumentException_ValueNotSupported()
   at System.Text.Json.Utf8JsonWriter.WriteNumberValue(Double value)
   at System.Text.Json.JsonSerializer.WriteDictionary[TProperty](JsonConverter`1 converter, JsonSerializerOptions options, WriteStackFrame& current, Utf8JsonWriter writer)
   at System.Text.Json.JsonSerializer.HandleDictionary(JsonClassInfo elementClassInfo, JsonSerializerOptions options, Utf8JsonWriter writer, WriteStack& state)
   at System.Text.Json.JsonSerializer.Write(Utf8JsonWriter writer, Int32 originalWriterDepth, Int32 flushThreshold, JsonSerializerOptions options, WriteStack& state)
   at System.Text.Json.JsonSerializer.WriteCore(Utf8JsonWriter writer, Object value, Type type, JsonSerializerOptions options)
   at System.Text.Json.JsonSerializer.WriteCore(PooledByteBufferWriter output, Object value, Type type, JsonSerializerOptions options)
   at System.Text.Json.JsonSerializer.WriteCoreString(Object value, Type type, JsonSerializerOptions options)
   at PQTest.Common.Commands.RunQueryTests(IEnumerable`1 queryTests, Configuration configuration) in C:\__w\1\s\Tools\PQTest\Common\Commands.cs:line 708
   at PQTest.Common.Commands.ConfigureMashupEnvironmentAndRun(Configuration configuration, Action action) in C:\__w\1\s\Tools\PQTest\Common\Commands.cs:line 584
   at PQTest.Console.Program.RunTestBatteryCommand(CommandLineApplication app, CommandLineApplication command, Action`1 invoke) in C:\__w\1\s\Tools\PQTest\Console\Program.cs:line 643
   at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args)
   at PQTest.Console.Program.Main(String[] args) in C:\__w\1\s\Tools\PQTest\Console\Program.cs:line 374
````

**Comment by mattmasson** [comment]
Don't do that.

**Comment by mattmasson** [comment]
Fixed in the 2.110.2 release of the SDK Tools nuget (which should be automatically downloaded by the preview VSIX).

**Comment by bgribaudo** [comment]
Thanks!
